keyboards/import/keys-Zyyy-punctuation.xml

```diff
@@ -2,36 +2,41 @@
 <!DOCTYPE keys SYSTEM "../dtd/ldmlKeyboard.dtd">
 <keys>
     <!-- General Symbols -->
+    <key id="amp" to="\u{0026}" />
     <key id="apos" to="'" />
-    <key id="bang" to="!" />
-    <key id="close-paren" to=")" />
-    <key id="double-quote" to="\u{0022}" />
-    <key id="equal" to="=" />
-    <key id="hash" to="#" />
-    <key id="hyphen" to="-" />
-    <key id="open-paren" to="(" />
-    <key id="umlaut" to="¨" />
-    <key id="underscore" to="_" />
     <key id="at" to="@" />
     <key id="backslash" to="\u{005C}" />
+    <key id="bang" to="!" />
     <key id="caret" to="^" />
     <key id="close-angle" to=">" />
     <key id="close-curly" to="}" />
+    <key id="close-paren" to=")" />
     <key id="close-square" to="]" />
     <key id="colon" to=":" />
+    <key id="comma" to="," />
+    <key id="double-quote" to="\u{0022}" />
+    <key id="equal" to="=" />
     <key id="grave" to="`" />
+    <key id="hash" to="#" />
+    <key id="hyphen" to="-" />
     <key id="not" to="¬" />
     <key id="open-angle" to="\u{003C}" />
     <key id="open-curly" to="{" />
+    <key id="open-paren" to="(" />
     <key id="open-square" to="[" />
+    <key id="percent" to="%" />
+    <key id="period" to="." />
     <key id="pipe" to="|" />
     <key id="plus" to="+" />
     <key id="question" to="?" />
     <key id="semi-colon" to=";" />
+    <key id="slash" to="/" />
     <key id="tilde" to="~" />
-    <key id="amp" to="\u{0026}" />
+    <key id="umlaut" to="¨" />
```

Comment:
I wonder about adding `umlaut` and `not` to this standard file -- I understand that they are on some European keyboards but they are outside the ascii range that all the other punctuation fits into, and may consistute a 'slippery slope' for adding more and more characters to this file? (Same could be argued for `pound` and even `euro` (these could be in a `keys-Zyyy-currency.xml` instead?).